src/main/java/com/google/sps/data/Instance.java

```diff
@@ -0,0 +1,88 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+import java.util.Optional;
+import com.google.sps.data.EntityConstants;
+
+/** Class that represents an instance. */
+public class Instance implements InstanceInterface {
+  
+  // The name of this instance.
+  private final String instanceName;
+
+  // The session associated with this instance.
```

Comment:
Add more to this comment. If the sessionId is not present, then the VM is not being used for a session